types/config.d.ts

```diff
@@ -16,13 +17,36 @@ declare global {
       export interface Json {
         extends?: 'lighthouse:default' | string | boolean;
         settings?: SharedFlagsSettings;
+        artifacts?: ArtifactJson[] | null;
```

Comment:
ah that's pleasant.  

I faintly remember some other hack where we allowed something to be expressed in JSON instead of read from a filepath. Ah, looks like it was --extra-headers.

nevermind carry on!